File: wireless/v1/sim/update-example-1/update-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

sim = @client.wireless.sims('DEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA')
  .update(
     callback_method: 'POST',
     callback_url: 'https://sim-manager.mycompany.com/sim-update-callback/AliceSmithSmartMeter',
     status: 'active'
   )

puts sim.unique_name

File: wireless/v1/sim/update-example-1/update-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Wireless.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var sim = SimResource.Update(
            callbackMethod: Twilio.Http.HttpMethod.Post,
            callbackUrl: new Uri("https://sim-manager.mycompany.com/sim-update-callback/AliceSmithSmartMeter"),
            status: SimResource.StatusEnum.Active,
            pathSid: "DEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
        );

        Console.WriteLine(sim.UniqueName);
    }
}
